libs/agno/agno/tools/bitbucket.py

```diff
@@ -0,0 +1,353 @@
+import base64
+import json
+import os
+from typing import Any, Dict, Optional, Union
+
+import requests
+
+from agno.tools import Toolkit
+from agno.utils.log import logger
+
+
+class BitbucketTools(Toolkit):
+    """A class that provides tools for interacting with the Bitbucket API."""
+
+    def __init__(
+        self,
+        server_url: Optional[str] = "api.bitbucket.org",
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+        token: Optional[str] = None,
+        api_version: Optional[str] = "2.0",
+    ):
+        """Initializes Bitbucket Tools.
+
+        Args:
+            server_url (str, optional): The Bitbucket server URL. Defaults to "api.bitbucket.org".
+            username (str, optional): The username to authenticate with. If not provided, it will take the value of `BITBUCKET_USERNAME` env variable.
+            password (str, optional): The password to authenticate with. If not provided, it will take the value of `BITBUCKET_PASSWORD` env variable..
+            token (str, optional): The token to authenticate with. If not provided, it will take the value of `BITBUCKET_TOKEN` env variable..
```

Comment:
```suggestion
            password (str, optional): The password to authenticate with. If not provided, it will take the value of `BITBUCKET_PASSWORD` env variable.
            token (str, optional): The token to authenticate with. If not provided, it will take the value of `BITBUCKET_TOKEN` env variable.
```